_posts/2017-12-23-crystal-0.24.1-released.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Crystal 0.24.1 released!
+summary: Crystal has a new development release.
+thumbnail: +
+author: straight-shoota
+release_notes: true
+---
+
+[Crystal 0.24.1](https://github.com/crystal-lang/crystal/releases/tag/0.24.1) has been released!
+
+As any release it includes numerous bugfixes, cool features and performance improvements - in [387 commits since 0.23.1](https://github.com/crystal-lang/crystal/compare/0.23.1...0.24.1).
+
+0.24.0 was a preview release so this is the canonical realease following 0.23.1.
+
+# Exciting Changes
+
+* With `ThinLTO`, LLVM 4.0 and above allows faster release builds.
+* `SecureRandom` is renamed to `Random::Secure` and `SecureRandom.uuid` is replaced by a new type: `UUID`.
+* `BigDecimal` offers arbitrary precision, exact decimal numbers, supplementing `BigInt` and `BigFloat` which all are required by `require "big"`.
+* `Set` instances can be used as condition for `case`, matching when the case variable is included in the set.
+* `Time::Format` more robustly supports parsing sub-second precision.
+* `Tempfile` supports custom extensions.
+* The docs generator outputs documentation as JSON which can be easily parsed and used for external applications. The output directory of the docs generator can be configured using `--output` flag.
+* Garbage collection can be disabled using compiler flag `-Dgc_none`.
```

Comment:
This is an implementation detail, and shouldn't be in these notes. It's not actually useful to anyone. (except @ysbaddaden as he implements a custom GC for crystal).

Reply:
I disagree, removing the GC is useful. For example a small lived programs may not care about leaking, since they'll most probably exit before the collector even runs.

Reply:
I think people will get the wrong impression. I've already seen people interpret "no gc" be "like rust somehow magically". GC is magic to a lot of people, I don't think advertising this is a good idea.